sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/kafka/BeamKafkaProtoTable.java

```diff
@@ -32,10 +34,10 @@
   "nullness" // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
 })
 public class BeamKafkaProtoTable extends BeamKafkaTable {
-  private final Class<?> protoClass;
+  private final Class<? extends Message> protoClass;
 
   public BeamKafkaProtoTable(
-      Schema messageSchema, String bootstrapServers, List<String> topics, Class<?> protoClass) {
+      Schema messageSchema, String bootstrapServers, List<String> topics, Class<? extends Message> protoClass) {
```

Comment:
This change breaks any clients that used to pass a mere `Class<?>` here!
Is this acceptable?

Reply:
reverted